src/utils/train.py

```diff
@@ -152,15 +151,22 @@ def train(
         evaluate_model(model, val_dataloader, device, sl, log=True, group="val")
 
     if test_dataloader is not None:
-        evaluate_model(model, test_dataloader, device, sl, log=True, group="test")
+        evaluate_model(model, test_dataloader, device, sl, log=True, group="test", no_average=config.no_average)
```

Comment:
I guess we also need to pass this option for validation holdout (lines 147 and 152)